### PR TITLE
First draft of learning contract

### PR DESCRIPTION
**Pull request by shakalee14**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+## Quality Rubric
+
+- [ ] Variables, functions, css classes, etc. are meaningfully named (no comments exist in code to explain functionality - the names serve that function)
+- [ ] Commit messages are concise and descriptive.
+- [ ] Each commit has a specific scope, and does not include unrelated code.
+- [ ] All features are added via pull requests.
+- [ ] Every pull request has a description summarizing the changes made.
+- [ ] Every pull request has been reviewed with [Prrr](http://prrr.apps.learnersguild.org/).
+- [ ] Code is sufficiently covered with a unit test suite
+- [ ] Displays nicely on a cell phone
```
Comment (deonna) [inline README.md]: Can you clarify 'nicely'? Should everything be contained with no horizontal scrolllbars, for example? Should there be a media query that shows a hamburger menu when the device height and width are smaller than some threshold?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Quality Rubric
+
+- [ ] Variables, functions, css classes, etc. are meaningfully named (no comments exist in code to explain functionality - the names serve that function)
+- [ ] Commit messages are concise and descriptive.
+- [ ] Each commit has a specific scope, and does not include unrelated code.
+- [ ] All features are added via pull requests.
+- [ ] Every pull request has a description summarizing the changes made.
+- [ ] Every pull request has been reviewed with [Prrr](http://prrr.apps.learnersguild.org/).
+- [ ] Code is sufficiently covered with a unit test suite
```
Comment (deonna) [inline README.md]: Do you have a code coverage %age? What is 'sufficient' in your eyes?

**Review by deonna — APPROVED** [review]
Good job on the learning contract -- see comments below.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+### Ethan Stark
+- [ ] Build a personal portfolio site showcasing web development projects
+- [ ] As a visitor, I can easily contact Ethan Stark via email or phone
+- [ ] As a visitor, I can navigate to related links: Github, project sites, LinkedIn
+- [ ] As a visitor, I can browse a gallery of web projects
+- [ ] As a visitor, I can click a web project and view additional details
+- [ ] Site is deployed to EthanStark.com
+
+
+## Quality Rubric
```
Comment (nodatall) [inline README.md:29]: I would add points to these items. Weight the items highly that you think will contribute most to your learning.

**Pull request merged**